test/statistics.jl

```diff
@@ -0,0 +1,16 @@
+@testset "Statistics" begin
+
+using CuArrays
+using Statistics
+
+@testset "std" begin
+    A = rand(10, 1, 2)
+    cuA = cu(A)
+
+    @test std(cuA[:, 1, 1]) ≈ std(A[:, 1, 1])
```

Comment:
runtests.jl defines a compare function that automatically constructs a CuArray, performs the method, and compares back on the CPU. so you just need `compare(std, rand(10, 1, 2))`.

Reply:
I found CuArrays uses `testf` which, in turn, depends on `compare`. This is probably what you want?